scalameta/semantic/shared/src/main/scala/scala/meta/internal/semantic/Meta.scala

```diff
@@ -0,0 +1,82 @@
+package scala.meta
+package internal
+package semantic
+
+import java.nio.charset.Charset
+import org.scalameta.data._
+import scala.{Seq => _}
+import scala.collection.immutable.Seq
+import scala.meta.io._
+import scala.meta.internal.semantic.{schema => s}
+import scala.meta.{semantic => m}
+import scala.meta.{Dialect => mDialect}
```

Comment:
What's the motivation for this rename? This gets shadowed by the `mDialect` extractor below.